config-overrides.js

```diff
@@ -1,18 +1,37 @@
+const CspHtmlWebpackPlugin = require('csp-html-webpack-plugin')
 const _ = require('lodash/fp')
 const rewireReactHotLoader = require('react-app-rewire-hot-loader')
-const { addBabelPlugin } = require('customize-cra')
+const { addBabelPlugin, addWebpackPlugin } = require('customize-cra')
 
 
 module.exports = {
   webpack(config, env) {
-    const newConfig = addBabelPlugin([
-      'prismjs', {
-        languages: ['bash', 'python'],
-        plugins: ['line-numbers'],
-        theme: 'default',
-        css: true
-      }
-    ])(config)
+    const newConfig = _.flow(
+      addBabelPlugin([
+        'prismjs', {
+          languages: ['bash', 'python'],
+          plugins: ['line-numbers'],
+          theme: 'default',
+          css: true
+        }
+      ]),
+      addWebpackPlugin(new CspHtmlWebpackPlugin({
+        // Per our security policy, 'script-src' cannot contain any of the options containing 'unsafe', e.g. 'unsafe-inline'
+        'script-src': [
+          '\'self\'',
+          'https://fast.appcues.com', // secondary script loaded by appcues
+          'https://us.jsagent.tcell.insight.rapid7.com', // secondary script loaded by tcell
+          'https://cdnjs.cloudflare.com' // libraries loaded by notebook preview
```

Comment:
Does this mean, "Any script can run if I can get it served from cdnjs?"

Reply:
Yes. However, it's important to note that in order to inject an arbitrary link to cdnjs, you'd need to already have an XSS vector, at which point it's probably moot. Ultimately we're relying on React's built-in XSS protection like we always have.